theano/configdefaults.py

```diff
@@ -1238,21 +1252,20 @@ def default_blas_ldflags():
                                           "mk2_rt"]])
 
         # Anaconda
-        if "Anaconda" in sys.version or "Continuum" in sys.version:
-            # If the "mkl-service" conda package (available
-            # through Python package "mkl") is installed and
+        if "Continuum Analytics" in sys.version:
```

Comment:
Should I add the comment regarding `maybe_add_to_os_environ_pathlist` to the 2 call-sites or to the doc string of the function instead?